tsl/test/expected/hypercore.out

```diff
@@ -138,17 +138,32 @@ WHERE time = '2022-06-01'::timestamptz;
 
 -- Create a new index on a compressed column
 CREATE INDEX ON readings (location);
--- Check that we error out on unsupported index types
+-- Run these checks as superuser to avoid permission errors. These
```

Comment:
Is it expected to have permission errors or is that also a problem?

Reply:
It is needed to create the index in the same schema as the chunk. I granted permissions to the default user at the beginning of the test instead to avoid confusions.